### ISSUE TITLE
fontconfig compilation

### ISSUE THREAD

**Issue by shawnlaffan** [StrawberryPerl/build-extlibs#7]
Below are some notes to get fontconfig compilation working.  

TLDR: fontconfig needs msys gperf and gettext-devel, and self-compiled fribidi.

### gettext

The build log includes this error which is resolved by installing gettext-devel from msys2.
```
######## gonna launch: autoreconf -fiv
autoreconf-2.71: export WARNINGS=
autoreconf-2.71: Entering directory '.'
autoreconf-2.71: running: autopoint --force
/bin/autopoint: line 495: /usr/share/gettext/archive.dir.tar.xz: No such file or directory
tar: This does not look like a tar archive
tar: gettext-0.19.8: Not found in archive
tar: Exiting with failure status due to previous errors
autopoint: *** infrastructure files for version 0.19.8 not found; this is autopoint from GNU gettext-tools 0.21
autopoint: *** Stop.
autoreconf-2.71: error: autopoint failed with exit status: 1
######## retval=FAILURE
```

### gperf

The configure script throws errors about gperf not being available, so also add it to the msys install.
e.g.:
```
_5034__/fontconfig-2.13.1.build.log:WARNING: 'gperf' is missing on your system.
```

The resultant fontconfig dll does not appear have extra baked-in dependencies 
on gperf or gettext so Strawberry should not need to compile them itself
(checked using ```objdump -x ./bin/libfontconfig-1__.dll | grep 'DLL Name'```).
Installation can be done via ```pacman``` in the build step, but should be added to the spbuild init script as well.  

```bash
pacman -Sy gperf gettext-devel 
```

## fribidi 

The build system raises an error about ```hb-ft.h: No such file or directory```.  

This seems to be associated with fribidi, so add it to the list of downloads and installs before fontconfig and harfbuzz.  

in 5034: 
```fribidi-1.0.12```

In sources.list:
```https://github.com/fribidi/fribidi/releases/download/v1.0.12/fribidi-1.0.12.tar.xz```

**Comment by shawnlaffan** [comment]
PR #10 merged, so closing
